### ISSUE TITLE
A11y: Markup lacks main landmark

### ISSUE THREAD

**Issue by dstorey** [smashingmagazine/redesign#155]
Screen reader users can navigate with landmarks to go straight to a particular part of the page. One of the most useful is the main landmark (main element in HTML). This is missing from all the pages. This is a different way of solving WCAG 2.4.1 (bypass blocks). When the main element is included, screen readers will include it in landmark navigation mode (unlike skip links) and provide a keyboard shortcut to go directly to main (narrator key + m in Narrator for example).

On this site the `section id="main"` element should be the main element. The current markup has two nested sections (the second with class="block", but this isn't really semantically a section in the document outline. If the element needs to be there for layout purposes, probably better it would be:

```
<main id="main>
    <div class="block"></div>
</main>
```

Rather than:

```
<section id="main>
    <section class="block"></section>
</section>
```
